src/components/AnnotationLineConnector/AnnotationLineConnector.js

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useContext, useState } from 'react';
+import NoteContext from 'components/Note/Context';
+import { useSelector, shallowEqual } from 'react-redux';
+import core from 'core';
+import selectors from 'selectors';
+import { createPortal } from 'react-dom';
+
+import './AnnotationLineConnector.scss';
+
+
+const LineConnectorPortal = ({ children }) => {
+  const mount = document.getElementById("line-connector-root");
+  const el = document.createElement("div");
+  el.setAttribute('data-element', 'annotationLineConnector');
```

Comment:
ah I was wondering why this component can be disabled without specifically hook into the `useSelector(state => isElementDisabled(...))` logic. Turns out if an element is disabled then `openElement` won't do anything, then this component won't be rendered because of the `isLineOpen` check.

I'm not sure if we should still hook into the isDisabled logic to make things more explicit, even though it's not necessary.